translate zh-tw and zh-hk

zehuaiWANG opened the pull request:
translate the zh-tw and zh-hw

Ref: #4792
Ref: #4734
Ref: #4763
Ref: #4806

I want to keep all Chinese versions consistent, so I refer to `zh` version to decide which ones are translated and which ones are not.

/assign @floreks  @maciaszczykm @shu-mutou @tanjunchen @hwdef @tengqm

zehuaiWANG commented:
In practice, some translations may not be accurate. I will modify it after zh version. Anyway, any discussion is welcome.

tanjunchen commented:
I'm not very familiar with traditional Chinese, sorry
unassign @tanjunchen

tanjunchen commented:
/unassign @tanjunchen

hwdef commented:
You said you wanted to ask @tanjunchen  to review your PR,can I understand that you don't want me to review your code?
so,
/unassign @hwdef

hwdef commented:
Only one suggestion, Singapore use Simplified Chinese.

zehuaiWANG commented:
@hwdef  I can apologize if I did something that didn't please you. Let's let go of the unpleasant things and make the dashboard better. As mentioned above, any discussion is welcome.

hwdef commented:
I want to make the dashboard better, too.
Unfortunately, I am not familiar with Traditional Chinese and Singaporean Chinese.
I also feel that you are not familiar with Singapore Chinese through this PR.

If you are not familiar, it is better not to translate, if you want to learn, you can see this link

https://zh.wikipedia.org/wiki/%E6%96%B0%E5%8A%A0%E5%9D%A1%E8%8F%AF%E8%AA%9E

Let bygones be bygones. We can work together.

hwdef commented:
I agree with you in zh-sg, I will add zh-sg with simplified Chinese in #4806 .
As for this PR, please be patient, someone will review it soon.
Hope you can enjoy your vacation too.

Thanks for your contribution.

zehuaiWANG commented:
OK, thank you. Have a good holiday !!!

zehuaiWANG commented:
BTW, I want to keep several different zh consistent, so I refer to your zh translation in this pr (for example, which words are translated and which are not). So don't merge / review the translation first. I will make some changes after you finish translating the zh version to make them as consistent as possible. What's your opinion? @hwdef

hwdef commented:
I agree with you.
When we determined how to translate through discussion.
I will modify zh to the best version.
After that, we can translate zh-hk zh-tw zh-sg and zh-cn.
so we should hold #4806 and this PR.

shu-mutou commented:
/hold
until the end of Chinese New Year and agreement of zh members.

zehuaiWANG commented:
thanks for @hwdef and @shu-mutou, have a good holiday ~~~

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -227,7 +227,7 @@
   </source>
         <target state="new">
-    Scale
+    規模
```
縮放

zehuaiWANG replied:
Thank you. I'll modify it later.

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -256,15 +256,15 @@
       <trans-unit id="0f5bc8e972fe992b070094b40e9105fdc30badee" datatype="html">
         <source>Trigger a <x id="INTERPOLATION" equiv-text="{{data.displayName}}"/></source>
-        <target state="new">Trigger a <x id="INTERPOLATION" equiv-text="{{data.displayName}}"/></target>
+        <target state="new"><x id="INTERPOLATION" equiv-text="{{data.displayName}}"/>觸發器</target>
```
```suggestion
        <target state="new">觸發<x id="INTERPOLATION" equiv-text="{{data.displayName}}"/></target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -275,7 +275,7 @@
   </source>
         <target state="new">
-    Trigger
+    觸發器
```
```suggestion
    觸發
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -640,103 +640,103 @@
       <trans-unit id="a062b45312698e8669cb644bff887f93778d334f" datatype="html">
         <source>Scale</source>
-        <target state="new">Scale</target>
+        <target state="new">規模</target>
```
```suggestion
        <target state="new">縮放</target>
```

zehuaiWANG replied:
Thank you. I'll modify it later.

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1160,23 +1160,23 @@
       <trans-unit id="2e6be334a2152f179a557167f98ce4459ff9a2f9" datatype="html">
         <source>Age</source>
-        <target state="new">Age</target>
+        <target state="new">經過的時間</target>
```
年龄？

zehuaiWANG replied:
这里表示的是workload被创建后经历的时间，也是已经创建了多久，所以年龄确实可以。有别的建议吗?

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1360,31 +1360,31 @@
       <trans-unit id="986668e81481d0fd0934b4ef56b4236ccb298712" datatype="html">
         <source>Conditions</source>
-        <target state="new">Conditions</target>
+        <target state="new">狀態</target>
```
条件

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1549,7 +1549,7 @@
         <source>Age
       </source>
-        <target state="new">Age
+        <target state="new">經過時間
```
年龄

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1618,7 +1618,7 @@
       <trans-unit id="6af40687e73f3e54515b94380a6463e6f8744ff2" datatype="html">
         <source>Custom Resource Definitions</source>
-        <target state="new">Custom Resource Definitions</target>
+        <target state="new">定義自定義資源</target>
```
```suggestion
        <target state="new">自定義資源的定義</target>
```

zehuaiWANG replied:
这里用“用戶自定義的資源”会不会更好一些？

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1746,7 +1746,7 @@
       <trans-unit id="6834fa6b43d1ecbdf147c48dd9c4d72f1484571d" datatype="html">
         <source>Source</source>
-        <target state="new">Source</target>
+        <target state="new">資源</target>
```
resource = 资源

source = 事件源

line 1741, event = 事件

zehuaiWANG replied:
看上去不错，我之后再check一下

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1878,39 +1878,39 @@
       <trans-unit id="d9465e327fed60c8566555ae63a51d8d24b4dcbe" datatype="html">
         <source>CPU limits (cores)</source>
-        <target state="new">CPU limits (cores)</target>
+        <target state="new">CPU 最高需求 (cores)</target>
```
```suggestion
        <target state="new">CPU 限制 (cores)</target>
```

zehuaiWANG replied:
限制是否不够明显，因为request和limit都有限制的意思，但request是最低，limit是最高。我突出这一点。

shu-mutou replied:
FYI: In Japanese translation, "上限" is used for "limits".

zehuaiWANG replied:
“上限” looks good！

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1878,39 +1878,39 @@
       <trans-unit id="90ae2a4959c979abc049f4652f59b85e993e16cd" datatype="html">
         <source>CPU requests (cores)</source>
-        <target state="new">CPU requests (cores)</target>
+        <target state="new">CPU 最低需求 (cores)</target>
```
```suggestion
        <target state="new">CPU 需求 (cores)</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1878,39 +1878,39 @@
       <trans-unit id="85550e95dc12c58e2e711dbebe89f3448eb66c72" datatype="html">
         <source>Memory requests (bytes)</source>
-        <target state="new">Memory requests (bytes)</target>
+        <target state="new">Memory 最低需求 (bytes)</target>
```
```suggestion
        <target state="new">Memory 請求 (bytes)</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -1878,39 +1878,39 @@
       <trans-unit id="69779fac7ee122da8c2ab05806baea7dec1a47f6" datatype="html">
         <source>Memory limits (bytes)</source>
-        <target state="new">Memory limits (bytes)</target>
+        <target state="new">Memory 最高需求 (bytes)</target>
```
```suggestion
        <target state="new">Memory 內存 (bytes)</target>
```

zehuaiWANG replied:
这里limits是最高需求的意思，跟request是最低需求一样，我想保持他，或者你们有更好的想法？

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2018,115 +1994,103 @@
       <trans-unit id="f2db0be1c93c93d5af51ed6d3b42abb184d07c08" datatype="html">
         <source>CPU Usage (cores)</source>
-        <target state="new">CPU Usage (cores)</target>
+        <target state="new">CPU 使用率 (cores)</target>
```
```suggestion
        <target state="new">CPU 使用 (cores)</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2018,115 +1994,103 @@
       <trans-unit id="b433be6f17bb6c614c6adb6a81912baae4a619b7" datatype="html">
         <source>Memory Usage (bytes)</source>
-        <target state="new">Memory Usage (bytes)</target>
+        <target state="new">内存使用 (bytes)</target>
```
```suggestion
        <target state="new">内存實用 (bytes)</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2178,15 +2142,15 @@
       <trans-unit id="63a988d71f146da942b6b97d3770b55472c33f08" datatype="html">
         <source>Claim</source>
-        <target state="new">Claim</target>
+        <target state="new">要求</target>
```
```suggestion
        <target state="new">申領</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2230,71 +2194,71 @@
       <trans-unit id="accc38331fffe0055910477b45e525173d64c661" datatype="html">
         <source>Verbs</source>
-        <target state="new">Verbs</target>
+        <target state="new">動作</target>
```
```suggestion
        <target state="new">動詞</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2306,23 +2270,23 @@
       <trans-unit id="6da72a4cb98aa86a034afc7c4134910590838fc5" datatype="html">
         <source>Default request</source>
-        <target state="new">Default request</target>
+        <target state="new">默認要求</target>
```
```suggestion
        <target state="new">默認請求</target>
```

hwdef replied:
这里联系上下文的话，用默认需求是不是更好

zehuaiWANG replied:
是的，我觉得默认需求/请求应该都是可以的，或者默认最低需求？这里的request有最低的意思，limit表示最高。

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -2667,7 +2631,7 @@
         <source>Custom Resource Definitions
       </source>
-        <target state="new">Custom Resource Definitions
+        <target state="new">定義自定義資源
```
```suggestion
        <target state="new">自定義資源的定義
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4686,7 +4634,7 @@
       <trans-unit id="5de9d226db382155f482a557b832da6d63108112" datatype="html">
         <source>Selector</source>
-        <target state="new">Selector</target>
+        <target state="new">選擇</target>
```
```suggestion
        <target state="new">選擇器</target>
```

zehuaiWANG replied:
Thank you. I'll modify it later.

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4802,159 +4750,159 @@
       <trans-unit id="f7038874dae1845bd604f1c69424ebddd668e845" datatype="html">
         <source>Max surge: </source>
-        <target state="new">Max surge: </target>
+        <target state="new">最大 surge: </target>
```
```suggestion
        <target state="new">最大替換數: </target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4802,159 +4750,159 @@
       <trans-unit id="c39576ee549298f6b7efa5fac84940493685ba3d" datatype="html">
         <source>Max unavailable: </source>
-        <target state="new">Max unavailable: </target>
+        <target state="new">最大不可用: </target>
```
```suggestion
        <target state="new">最大不可用数: </target>
```

zehuaiWANG replied:
Thank you. I'll modify it later.

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4802,159 +4750,159 @@
       <trans-unit id="248e61c25d07d6906e7caa95e59f6d9557136b47" datatype="html">
         <source>Max surge</source>
-        <target state="new">Max surge</target>
+        <target state="new">最大 surge</target>
```
```suggestion
        <target state="new">最大替換數</target>
```

chenrui333 commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4802,159 +4750,159 @@
       <trans-unit id="3f43eb0d43c0d3b9cb89ff03e01bd2704e596279" datatype="html">
         <source>Max unavailable</source>
-        <target state="new">Max unavailable</target>
+        <target state="new">最大不可用</target>
```
```suggestion
        <target state="new">最大不可用数</target>
```

zehuaiWANG replied:
Thank you. I'll modify it later.

zehuaiWANG commented:
Thanks for @chenrui333 @hwdef‘s review. Most of them look good and helpful. I'll modify them later. A small part of them may need further discussion. I'm also not sure now, I will see how official documents are translated later. Thank you again. Have a good holiday !!!

shu-mutou commented:
Please check #4814. It reconstructs Chinese locales and realizes maintaining only Simplifiled Chinese `zh-hans` and Traditional Chinese `zh-hant`.
* `zh`, `zh-cn` and `zh-sg` are linked from `zh-hans`.
* `zh-tw`  and `zh-hk` are linked from `zh-hant`.

hwdef commented:
I think we can continue to do this work.
If there is anything I can help, please @ me.

shu-mutou commented:
Could you rebase to include recent additions about `Horizontal Pod Autoscaler` ?

zehuaiWANG commented:
@hwdef @chenrui333 hi~ I had modified most of them, if you have sometimes, could you help to review it again and check if there are any inconsistencies with the zh version. And after zh version had added the `Horizontal Pod Autoscaler` part, please let me know, I will try to do it after the zh version updating.

zehuaiWANG commented:
In addition, due to the current outbreak of novel coronavirus (2019-nCoV) .  I am still on vacation, so the response may be slow. Please feel free to do any update, if it is urgent. @hwdef

hwdef commented:
@zehuaiWANG 
thanks for your contributions.
Please take care of yourself, there are many infected people in Shenzhen.

zehuaiWANG commented:
ahh, thank you, I'm not in Shenzhen recently.   :)

zehuaiWANG commented:
@shu-mutou @hwdef I have updated the `HPA` part, please check to see if there are any problems.

hwdef commented:
Roughly reading, there is no problem. Maybe we can merge this PR first. If there are problems later, we can change it.
Is it OK to do this?
@tanjunchen @chenrui333 @tengqm

hwdef commented:
/lgtm
/approve

shu-mutou commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4974,55 +4922,47 @@
-          <context context-type="linenumber">284</context>
-        </context-group>
-      </trans-unit>
       <trans-unit id="5413cb2f145c06134be125af0b8900fa43178672" datatype="html">
```
ditto

shu-mutou replied:
ditto

shu-mutou commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4974,55 +4922,47 @@
-          <context context-type="linenumber">284</context>
-        </context-group>
-      </trans-unit>
       <trans-unit id="5413cb2f145c06134be125af0b8900fa43178672" datatype="html">
```
Do not remove this. This is singular and different from above `Horizontal Pod Autoscalers` (plural).

shu-mutou commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -4974,55 +4974,55 @@
+        <target state="new">Pod 自動水平伸縮</target>  
+        <context-group purpose="location">  
+          <context context-type="sourcefile">../src/app/frontend/resource/workloads/deployment/detail/template.html</context> 
+          <context context-type="linenumber">284</context>  
```
there are trailing spaces in each line. could you remove them?

shu-mutou commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4974,55 +4974,55 @@
+        <context-group purpose="location">  
+          <context context-type="sourcefile">../src/app/frontend/resource/workloads/deployment/detail/template.html</context> 
+          <context context-type="linenumber">284</context>  
+        </context-group>  
```
ditto

shu-mutou commented:
/hold cancel
@zehuaiWANG Thank you so much!

zehuaiWANG commented on i18n/zh-hk/messages.zh-hk.xlf:
```diff
@@ -5070,8 +5070,7 @@
     </source>
         <target state="new">
-      Global settings are stored in config map, so all of them are applied for every instance of the
-      app.
```
Could you help to check why this is in a new line? @shu-mutou

shu-mutou replied:
You can find original context from following `context` tag, i.e. line 5077.
It points https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/settings/global/template.html#L23-L26

zehuaiWANG replied:
ahh, I checked it on the UI page, they are placed in the `p` label on the same line, and it looks like an optional line break. In any case, he will not cause problems.

hwdef commented:
please rebase your pr and cleanup commit information

hwdef commented:
![image](https://user-images.githubusercontent.com/13084946/73728884-15bef700-476f-11ea-9958-8b453b553924.png)

This doesn't look good

hwdef commented:
/hold

zehuaiWANG commented:
I will rebase it at the end of all reviews, thank you for your reminder. @hwdef

zehuaiWANG commented:
I have responded to most of the comments and modified pr based on the comments, please help to review if there are any problem or any inconsistencies with other `zh` versions, and in addition, I had rebased it. @hwdef

hwdef commented:
I think this PR can be merged. But I want to wait for other reviewer to review this again.

hwdef commented:
/hold cancel

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2038,95 +2038,95 @@
       <trans-unit id="b433be6f17bb6c614c6adb6a81912baae4a619b7" datatype="html">
         <source>Memory Usage (bytes)</source>
-        <target state="new">Memory Usage (bytes)</target>
+        <target state="new">内存實用 (bytes)</target>
```
記憶體用量

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2306,23 +2306,23 @@
       <trans-unit id="e4bc0a718dd945e7242e230772c371d963128d11" datatype="html">
         <source>Resource type</source>
-        <target state="new">Resource type</target>
+        <target state="new">资源類別</target>
```
資源類別

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2382,7 +2382,7 @@
       <trans-unit id="8fcda5141768dbab8069cf689584f4ec284828d3" datatype="html">
         <source>Cluster IP</source>
-        <target state="new">Cluster IP</target>
+        <target state="new">集群 IP</target>
```
叢集 IP

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2427,7 +2427,7 @@
         <source>Cluster
       </source>
-        <target state="new">Cluster
+        <target state="new">集群
```
叢集

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2765,7 +2765,7 @@
         <source>Sign in
   </source>
-        <target state="new">Sign in
+        <target state="new">登錄
```
登入

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2775,7 +2775,7 @@
         <source>Sign out
   </source>
-        <target state="new">Sign out
+        <target state="new">注銷
```
登出

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2794,15 +2794,15 @@
       <trans-unit id="f4d1dd59b039ad818d9da7e29a773e10e41d9821" datatype="html">
         <source>Cluster</source>
-        <target state="new">Cluster</target>
+        <target state="new">集群</target>
```
叢集

zehuaiWANG commented:
@oneoneonepig most of them look good~ Thank you very much !!!
/ hold hold this pr before @oneoneonepig review.

zehuaiWANG commented:
/hold

zehuaiWANG commented:
@oneoneonepig hi~ Could you help to review the `tw` part if you have some time? I will modify them after your review. Thanks a lot.

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3671,7 +3671,7 @@
           </source>
         <target state="new">
-            Memory requirement must be given as a valid number.
+            Memory 最低需求必须是一个有效的数字。
```
Memory 最低需求必須是一個有效的數字。

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="e34bb7f0c94de7773e15d02b03df26f254a7142d" datatype="html">
         <source>You can specify minimum CPU and memory requirements for the container.</source>
-        <target state="new">You can specify minimum CPU and memory requirements for the container.</target>
+        <target state="new">您可以指定容器的最低 CPU 和内存 requirements。</target>
```
您可以指定容器的最低 CPU 和記憶體需求。

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
           <context context-type="sourcefile">../src/app/frontend/create/from/form/template.html</context>
           <context context-type="linenumber">291</context>
         </context-group>
       </trans-unit>
```
執行命令

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="f673c95db1b46263de2e28a17989d58b63b7033b" datatype="html">
         <source>Run command arguments</source>
-        <target state="new">Run command arguments</target>
+        <target state="new">运行命令参数</target>
```
執行命令參數

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="7f64de7337ceb8f09ec7db1778676b02f07a73df" datatype="html">
         <source>By default, your containers run the selected image's default entrypoint command. You can use the command options to override the default.</source>
-        <target state="new">By default, your containers run the selected image's default entrypoint command. You can use the command options to override the default.</target>
+        <target state="new">默认情况下，容器运行所选镜像的默认 entrypoint command。 您可以使用命令选项覆盖默认值。</target>
```
默認情况下，容器運行所選映像檔的默認 entrypoint 命令。您可以使用命令選項覆蓋默認值。

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="c6da8aedaa5d3cd0687973b159947e1f22d05b42" datatype="html">
         <source>Run as privileged</source>
-        <target state="new">Run as privileged</target>
+        <target state="new">以特权身份运行</target>
```
以特權身份運行

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="02d6959168cab2d5d4a1f513f7e74f1a9c718edf" datatype="html">
         <source>Processes in privileged containers are equivalent to processes running as root on the host.</source>
-        <target state="new">Processes in privileged containers are equivalent to processes running as root on the host.</target>
+        <target state="new">特权容器中的进程等同于在主机上以root身份运行的进程。</target>
```
特權容器中的程序等同於在主機上以 root 身份執行的程序。

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3680,55 +3680,55 @@
       <trans-unit id="d2f13c0a4c3a1e77e3cb724cafab9a0480be0ff3" datatype="html">
         <source>Environment variables available for use in the container. Values can reference other variables using $(VAR_NAME) syntax.</source>
-        <target state="new">Environment variables available for use in the container. Values can reference other variables using $(VAR_NAME) syntax.</target>
+        <target state="new">可在容器中使用的环境变量。值可以使用 $(VAR_NAME) 语法引用其他变量。</target>
```
可在容器中使用的環境變數。值可以使用 $(VAR_NAME) 語法引用其他變數

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3823,7 +3823,7 @@
 </source>
         <target state="new">
-  Upload
+            上传
```
上傳

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3890,7 +3890,7 @@
       <trans-unit id="4703f18f9d5b6cd5c3c1f8a5d9677ee0ffb29c19" datatype="html">
         <source>Environment variables</source>
-        <target state="new">Environment variables</target>
+        <target state="new">環境變量</target>
```
環境變數

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3910,7 +3910,7 @@
       </trans-unit>
```
變數名必須是有效的 C 標識符

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4170,15 +4170,15 @@
       <trans-unit id="1baabee7b86d4d0935657f254c3b17b5fe4d1257" datatype="html">
         <source>Logs from</source>
-        <target state="new">Logs from</target>
+        <target state="new">日志</target>
```
日誌

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4194,7 +4194,7 @@
       <trans-unit id="bdf43b20b7101e83458a69a1e5e2008557feca1f" datatype="html">
         <source>Download logs</source>
-        <target state="new">Download logs</target>
+        <target state="new">下載日志</target>
```
下載日誌

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4205,7 +4205,7 @@
         </source>
         <target state="new">
-          Logs from <x id="START_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;kd-date>"/><x id="CLOSE_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;/kd-date>"/> to <x id="START_TAG_KD-DATE_1" ctype="x-kd-date" equiv-text="&lt;kd-date>"/><x id="CLOSE_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;/kd-date>"/> UTC
+          日志來自 <x id="START_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;kd-date>"/><x id="CLOSE_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;/kd-date>"/> to <x id="START_TAG_KD-DATE_1" ctype="x-kd-date" equiv-text="&lt;kd-date>"/><x id="CLOSE_TAG_KD-DATE" ctype="x-kd-date" equiv-text="&lt;/kd-date>"/> UTC
```
日誌來自

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4254,15 +4254,15 @@
       <trans-unit id="3de87d94cff5d685043f035ac20e6d4521f8edf2" datatype="html">
         <source>Show previous logs</source>
-        <target state="new">Show previous logs</target>
+        <target state="new">顯示以前的日志</target>
```
日誌

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4278,127 +4278,127 @@
       <trans-unit id="b129f142540fcf057f58e5ffc3a4bd1aae6f074b" datatype="html">
         <source>System information</source>
-        <target state="new">System information</target>
+        <target state="new">系統信息</target>
```
系統訊息

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4278,127 +4278,127 @@
       <trans-unit id="0125b821613421311435ad931bc1d25d45a256df" datatype="html">
         <source>Boot ID</source>
-        <target state="new">Boot ID</target>
+        <target state="new">啓動 ID</target>
```
啟動 ID

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4278,127 +4278,127 @@
           <context context-type="sourcefile">../src/app/frontend/resource/cluster/node/detail/template.html</context>
           <context context-type="linenumber">90</context>
         </context-group>
       </trans-unit>
```
核心版本

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4278,127 +4278,127 @@
       <trans-unit id="b398d1a80e8ad17d3e6cb69370e5c49613e16ec9" datatype="html">
         <source>OS Image</source>
-        <target state="new">OS Image</target>
+        <target state="new">操作系統鏡像</target>
```
操作系統映像檔

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -4494,7 +4494,7 @@
       <trans-unit id="083338c1e13796fd0259d89493ec7b75be0d14d0" datatype="html">
         <source>Read only</source>
-        <target state="new">Read only</target>
+        <target state="new">只讀</target>
```
唯讀

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -5080,7 +5079,7 @@
       <trans-unit id="2509fab3be38702830979ab6368f37c52bfb8f80" datatype="html">
         <source>Cluster name</source>
-        <target state="new">Cluster name</target>
+        <target state="new">集群名稱</target>
```
叢集名稱

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -5092,55 +5091,55 @@
       <trans-unit id="409222992640ac7a22c9f4d8d1dd19bf0109bafa" datatype="html">
         <source>Cluster name appears in the browser window title if it is set.</source>
-        <target state="new">Cluster name appears in the browser window title if it is set.</target>
+        <target state="new">如果已設置， 則集群名字將顯示在瀏覽器窗口標題中。</target>
```
如果已設置， 則叢集名字將顯示在瀏覽器標題中。

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -5092,55 +5091,55 @@
       <trans-unit id="1bf6e9fbb0027605b8c66e07e51ba28ff29ef67f" datatype="html">
         <source>Logs auto-refresh time interval</source>
-        <target state="new">Logs auto-refresh time interval</target>
+        <target state="new">日志自動刷新時間間隔</target>
```
日誌

oneoneonepig commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -5092,55 +5091,55 @@
       <trans-unit id="02458bd7e54108bc7ac4f35fc96aeb3e422fa471" datatype="html">
         <source>Number of seconds between every auto-refresh of logs.</source>
-        <target state="new">Number of seconds between every auto-refresh of logs.</target>
+        <target state="new">每次自動刷新日志的間隔秒數。</target>
```
日誌

zehuaiWANG commented:
@oneoneonepig Thank you very much for your contribution. I will modify them later. Could you help to submit a PR after that to modify the `tw` version again if you find any inaccuracies?  :)

oneoneonepig commented:
Sure !

zehuaiWANG commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -2885,7 +2885,7 @@
               </source>
         <target state="new">
-                Make sure that support for basic authentication is enabled in the cluster. To find out more about how to configure basic authentication, please refer to the <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>Authenticating<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> and <x id="START_LINK_1" ctype="x-a" equiv-text="&lt;a>"/>ABAC Mode<x id="CLOSE_LINK" ctype="x-a" equiv-te ...
+                確保在集群中啓用了對基本身份驗證的支持。 要了解有關如何配置基本身份驗證的詳情，请參閲 <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>Authenticating<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> 和 <x id="START_LINK_1" ctype="x-a" equiv-text="&lt;a>"/>ABAC Mode<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> 部分.
```
支援？支持？我覺得這裏可以維持原樣，來保持和其他版本的一致。

zehuaiWANG commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3465,7 +3465,7 @@
         Number of pods must be a positive integer
```
pod 的數量必須是整数
ps: source裏面的内容不需要被翻譯

zehuaiWANG commented on i18n/zh-tw/messages.zh-tw.xlf:
```diff
@@ -3486,23 +3486,23 @@
       <trans-unit id="1b71dbca4d8c341a7a9be7f8ce266494a5ed3827" datatype="html">
         <source>A Deployment will be created to maintain the desired number of pods across your cluster.</source>
-        <target state="new">A Deployment will be created to maintain the desired number of pods across your cluster.</target>
+        <target state="new">Deployment 将跨集群创建 pod 以维护所需数量。</target>
```
Deployment 将被創建來管理一定數量的 pod 在你的叢集中。

zehuaiWANG commented:
/hold cancel

zehuaiWANG commented:
@hwdef I had modify most of them, I think it may looks good now. I wonder if we can merge this PR first, and if there are problems later, we can open a new one to modify it.
What more, thank you for your contribution. @hwdef @shu-mutou @chenrui333 @oneoneonepig

hwdef commented:
@zehuaiWANG  Thanks for your contributions. You did a great job!

/lgtm
/approve
